cp3pt0-deployment/setup_tenant.sh

```diff
@@ -464,8 +464,12 @@ EOF
     
         # Check if the patch was successful
         if [[ $? -eq 0 ]]; then
-            success "Successfully patched CommonService CR in ${OPERATOR_NS}"
-            break
+            checkOperatorNS=$(${OC} get commonservice common-service -n ${OPERATOR_NS} -o yaml | yq '.spec.operatorNamespace=="'${OPERATOR_NS}'"')
+            checkServicesNS=$(${OC} get commonservice common-service -n ${OPERATOR_NS} -o yaml | yq '.servicesNamespace=="'${SERVICES_NS}'"')
```

Comment:
```suggestion
            checkServicesNS=$(${OC} get commonservice common-service -n ${OPERATOR_NS} -o yaml | yq '.spec.servicesNamespace=="'${SERVICES_NS}'"')
```

Reply:
Could we cover some failure cases on tests?

Fr example, the `servicesNamespace` in the CS CR is not desired or this field is pruned by k8s APIServer(Manually edit CS CRD to remove `preserve-unknown-fields` marker and remove `servicesNamespace` field).